integration/jest-integration-test.config.js

```diff
@@ -0,0 +1,4 @@
+export default {
```

Comment:
can we name this config `jest.config.js` within here?  Since its scoped to this folder, it seems superfluous to add `-integration-test` to it as well.